wiki/Glossary/en.md

```diff
@@ -882,7 +901,7 @@ The universal offset applies to all songs used in osu! and its value differs acc
 
 ### upppy
 
-upppy is a fast and easy-to-use uploading service coded by peppy. If you want to share a file with the osu!community, forget about depositing it on Rapidshare, Megaupload, ImageShack, etc., just put it on upppy and everyone is happy! :)
+A fast and easy-to-use uploading service coded by peppy. If you want to share a file with the osu! community, forget about depositing it on Rapidshare, Megaupload, ImageShack, etc., just put it on upppy and everyone is happy! :)
 
 For more information about upppy, see [upppy!](https://up.ppy.sh) **(NSFW: there may be a few stray images that may contain R-18 work)**.
 
```

Comment:
for lines between 914 and 918
```
- The specific [beatmap](/wiki/beatmap) of the [beatmapset](#beatmapset).
  - This term can be used in place of beatmap to prevent disambiguation.
- The version of a skin. (See [skin.ini](/wiki/Skinning/skin.ini#versions) for more details.)